discretisedfield/tests/test_field.py

```diff
@@ -1967,6 +1967,67 @@ def value_fun(point):
         f.grad
 
 
+def test_div(valid_mesh):
+    nvdim = valid_mesh.region.ndim
+    vdims = ["v" + d for d in valid_mesh.region.dims]  # ensure unique vdims
+    vdim_mapping = dict(zip(vdims, valid_mesh.region.dims))
+    f = df.Field(valid_mesh, nvdim=nvdim, vdims=vdims, vdim_mapping=vdim_mapping)
+
+    assert isinstance(f.div, df.Field)
+    assert np.allclose(f.div.mean(), 0)
+
+    # f(x, y, z) = (x, y, z) -> div(f) = 1 + 1 + 1
+    def value_fun(point):
```

Comment:
Maybe directly use `valid_mesh.coordinate_field` as value? But I am not sure since the present form is more readable. You take a call.

Reply:
Nice!!